frontend/src/App.tsx

```diff
@@ -603,8 +570,6 @@ function validateOrFixPlayerName(name: string): string {
   return name;
 }
 
-function validateOrFixEloId(eloId: string): string {
-  if (eloId.length < minEloIdLen || eloId.length > maxEloIdLen)
-    return randEloId(maxEloIdLen);
-  return eloId;
+export function checkIsLoggedIn(idToken: string): boolean {
```

Comment:
Some comments:
1. Do not export functions from App.tsx, as it is the highest-level component. 
2. I think this function could be in the same file that exports auth0Prefix (maybe you don't need to export auth0Prefix anymore if you move this function there).